shared/chat/conversation/messages/unfurl/unfurl-list/index.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import * as React from 'react'
+import * as RPCChatTypes from '../../../../../constants/types/rpc-chat-gen'
+import * as Styles from '../../../../../styles'
+import {Box2} from '../../../../../common-adapters/index'
+import UnfurlGeneric from '../generic/container'
+
+export type UnfurlListItem = {
+  unfurl: RPCChatTypes.UnfurlDisplay,
+  url: string,
+  onClose: () => void,
+}
+
+export type ListProps = {
+  unfurls: Array<UnfurlListItem>,
+}
+
+export type UnfurlProps = {
+  unfurl: RPCChatTypes.UnfurlDisplay,
+  onClose: () => void,
+}
+
+class Unfurl extends React.PureComponent<UnfurlProps> {
+  render() {
+    switch (this.props.unfurl.unfurlType) {
+      case RPCChatTypes.unfurlUnfurlType.generic:
+        return this.props.unfurl.generic ? (
+          <UnfurlGeneric
```

Comment:
I guess, not sure what that really buys us.

Reply:
Hmm actually I see, let me play with this

Reply:
We can discuss, but I think this is likely what you want here (what I currently have). If you want to render a single unfurl, you are going to have the unfurl already handy, so might as well just pass props directly, instead of passing in something like a URL prop and having to pull it out again in connect.